### PR TITLE
perlclass: improve wording

### PR DESCRIPTION
**Pull request by mauke**
At least I think it's an improvement.

### CODE REVIEW

File: pod/perlclass.pod [hunk 1]
```diff
@@ -57,9 +58,9 @@ the scope of current package:
 
     class NAME VERSION : ATTRIBUTES...;
 
-The C<class> keyword declares a new package which is intended to be a class.
-All other keywords from the C<class> feature should be used in scope of this
-declaration.
+The C<class> keyword declares a new package that is intended to be a class.
```
Comment (jkeenan) [inline pod/perlclass.pod]: Is there some point in the documentation where we explicitly distinguish between `package` and `class`?  (If so, perhaps we should refer to it here.  If not, then let's add something here.)

File: pod/perlclass.pod [hunk 2]
```diff
@@ -270,22 +271,40 @@ named C<new> and is invoked like a method call on the class name:
...
-populated where possible.
+During object construction, class fields are looked up in the C<%arguments>
+hash and populated where possible.
 
 =head2 Adjustment
 
-Object adjustment can be performed during the construction to run user-defined
-code. It is done with the help of C<ADJUST> blocks, which are called in order
-of declaration.
+Object adjustment is way to run arbitrary user-defined code during object
```
Comment (jkeenan) [inline pod/perlclass.pod]: should be `is a way`

**Review by khwilliamson — APPROVED** [review]

**Comment by khwilliamson** [comment]
My approval is based on resolving @jkeenan 's comments

File: pod/perlclass.pod (hunk 1 above)
Comment (mauke) [reply pod/perlclass.pod]: Isn't that what the "History" section is about? There's more in perlmod ("Packages", "Perl Classes"), but I'm not sure how to reference that here.

**Review by leonerd — APPROVED** [review]
LGTM

File: pod/perlclass.pod (hunk 1 above)
Comment (mauke) [reply pod/perlclass.pod]: I've now added some links to perlfunc/bless, perlmod, and perlobj to the "History" section. In particular, perlmod explains how packages work in general. Do you think this provides enough background information or should we add something to the description of the `class` keyword, too?

**Pull request merged**